fe1-web/src/core/components/QrCodeScanner.tsx

```diff
@@ -57,62 +57,47 @@ const styles = StyleSheet.create({
  * (2022-12-05, Tyratox) Issue to track this: https://github.com/dedis/popstellar/issues/1306
  */
 const QrCodeScanner = ({ showCamera, children, handleScan }: IPropTypes) => {
-  const [permission, requestPermission] = Camera.useCameraPermissions();
+  // On future releases this might be changed to allow switching cameras. But for now this is not working. (2023-06-03)
```

Comment:
I would add your username in the comment too, this was the format we usually used in the past. Is a bit quicker to see who put the comment than using git blame :)

```suggestion
  // On future releases this might be changed to allow switching cameras. But for now this is not working. (2023-06-03, MeKHell)
```